smibhid/README.md

```diff
@@ -53,6 +54,8 @@ Copy the files from the smibhib folder into the root of a Pico W running Micropy
 - Set the space state poll frequency in seconds (>= 5), set to 0 to disable the state poll
 - Configure the space open relay pin if required or else set to None, also choose if space open sets pin high or low
 
+If you miss any configuration options, a default will be applied, an error output in the log detailing the configuration item missed including the default value configured and if connected, an error displayed on displays.
```

Comment:
If the change to the log message in the code is done, this will need to be updated